### ISSUE TITLE
how to connect

### ISSUE THREAD

**Issue by muescha** [simulieren/gatsby-starter-framer-x#2]
as newbee to framer x:

maybe a link to doc/tutorial "how to connect with framer x" would be nice

**Comment by simulieren** [comment]
Yes, I will write it down in the `README.md` as *Getting started with Framer X*.

**Comment by simulieren** [comment]
@muescha I just added a Getting Started section, please let me know if something needs further explanation. :)

**Comment by muescha** [comment]
framer x is an interesting tool

maybe you like also write a doc or blog about this in combination with gatsby (ask [@gatsbyjs/docs](https://github.com/orgs/gatsbyjs/teams/docs) team if there is an interest)

**Comment by simulieren** [comment]
@muescha Yeah, I also think that way. I'm just getting into it. 
I will do that, thank you for the suggestion! :)
